src/page-modules/contact/ticket-control-and-fee/index.tsx

```diff
@@ -22,10 +31,78 @@ export const TicketControlAndFeeContent = (
   };
 
   return (
-    <section>
-      <form onSubmit={onSubmit}>
-        <button type="submit">Send</button>
-      </form>
-    </section>
+    <div>
+      <SectionCard title={PageText.Contact.ticketControl.title}>
+        <FormSelector state={state} send={send} />
+      </SectionCard>
+      {state.matches('agreement1') && (
```

Comment:
Should this be `state.matches('feeComplaintForm')` ?

Reply:
No, because the first agreement should initially be displayed when choosing this option. But, the naming should probably be updated to be less confusing.